tensorflow/core/ops/array_ops.cc

```diff
@@ -2344,6 +2362,26 @@ REGISTER_OP("Slice")
         c->set_output(0, c->MakeShape(dims));
         return Status::OK();
       } else {
```

Comment:
there is an earlier comment // NOTE(mrry): We can't use `MakeShapeFromShapeTensor` for `sizes` because  ...

can you remove that comment, or say that it is done now if the sizes_value is null?

Reply:
Done.